src/http-api/index.js

```diff
@@ -23,6 +26,11 @@ function HttpApi (repo, config) {
   this.log = debug('jsipfs:http-api')
   this.log.error = debug('jsipfs:http-api:error')
 
+  // Setup debug metrics collection
+  const collectDefaultMetrics = prometheusClient.collectDefaultMetrics
+  collectDefaultMetrics({ timeout: 5000 })
+  prometheusGcStats(prometheusClient.register)()
```

Comment:
Sure, that makes sense, although it makes the usage different from go-ipfs who exposes it by default. But that makes sense, I'll move the dependencies to optionalDependencies as well then.